src/main/java/appeng/items/tools/powered/ToolWirelessTerminal.java

```diff
@@ -86,18 +156,15 @@ public void addCheckedInformation( final ItemStack stack, final EntityPlayer pla
 
 				if( encKey == null || encKey.isEmpty() )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this if-then-else statement by a single return statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1126)